src/kirby/components/card/card.component.html

```diff
@@ -1,4 +1,4 @@
-<div [ngClass]="colortype">
+<div [ngClass]="colortype" (click)="select.emit()">
```

Comment:
I think it should be a @HostListener instead, so we bind the click event to the host-tag (this wrapper div should be removed)